tests/utils.ts

```diff
@@ -89,6 +89,9 @@ export async function openTable(
 
   if (waitForLoadFinished) {
     await expect(page.locator('.iris-grid-panel')).toHaveCount(panelCount + 1);
+    await expect(page.locator('.iris-grid-panel .loading-spinner')).toHaveCount(
+      0
+    );
     await expect(
       page.locator('.iris-grid .iris-grid-loading-status')
     ).toHaveCount(0);
```

Comment:
You still need to make a positive assertion here that content is available of some format. Just the spinners being gone could flake out if there is nothing rendered as these assertions are always true even if there's no grid on the page